source/_components/binary_sensor.template.markdown

```diff
@@ -58,6 +64,11 @@ sensors:
           description: The sensor is `on` if the template evaluates as `True` and `off` otherwise. The actual appearance in the frontend (`Open`/`Closed`, `Detected`/`Clear` etc) depends on the sensor’s device_class value
           required: true
           type: template
+        availability_template:
+          description: "Defines a template to get the `available` state of the component. If the template returns `true` the device is `available`. If the template returns any other value, the device will be `unavailable`. If `availability_template` is not configured, the component will always be `available`"
```

Comment:
Remove the quotes around the entire description.

Reply:
Will do. I'll have to update this in all 8 PRs.